translator/src/main/scala/io/kaitai/struct/testtranslator/TestTranslator.scala

```diff
@@ -82,7 +82,7 @@ class TestTranslator(options: CLIOptions) {
         endian = None,
         bitEndian = None,
         encoding = None,
-        forceDebug = false,
+        forceDebug = debug,
```

Comment:
I wanted to name the new property as `auto-read` (with default to `true`), but because test is created using this code, more honestly to name it `debug`